go-app.js

```diff
@@ -87,10 +87,10 @@ go.utils = function () {
         body: body
       })
       .then(function (response) {
-        console.log(response);
+        im.log(response);
```

Comment:
Logging returns a promise, so you might want `return im.log(response).thenResolve(true);` here (and `.thenResolve(false)` in the other case).